sdk/digitaltwins/azure-digitaltwins-core/src/samples/java/com/azure/digitaltwins/core/ComponentSyncSamples.java

```diff
@@ -145,6 +145,7 @@ public static void runComponentSample() throws JsonProcessingException {
         ConsoleLogger.print("Retrieved component for digital twin " + basicDigitalTwinId + " :");
         for (String key : getComponentResponse.getContents().keySet()) {
             ConsoleLogger.print("\t" + key + " : " + getComponentResponse.getContents().get(key));
+            ConsoleLogger.print("\t\tLast updated on: " + getComponentResponse.getMetadata().get(key).getLastUpdatedOnOffsetDateTime());
```

Comment:
I added this to the component sample to demonstrate getting this property metadata field

Reply:
good catch !

Reply:
I came across the same issue when I was running the samples to verify JsonPatchDocument